src/main/java/flaxbeard/thaumicexploration/item/ItemFoodTalisman.java

```diff
@@ -26,11 +26,13 @@ public class ItemFoodTalisman extends Item {
 
     public static List<String> foodBlacklist = new ArrayList<String>();
     public static Map<String, Boolean> foodCache = new HashMap<String, Boolean>();
+    private final int MAX_HEAL_SIZE_TALISMAN = 1000;
+    private final int MAX_SAT_SIZE_TALISMAN = 1000;
```

Comment:
values while bigger may be a bit too high, need second opinion

Reply:
unstable ingots axe restores hunger without wasting anything, while the amulet wastes food and transfers its satiety to the player. I think whatever number you put, it will not cheat compared to the axe